README.md

```diff
@@ -0,0 +1,34 @@
+
+# DPM360
+Repository for Disease Progression Modeling workbench 360
+
+Overview and YouTube demonstration are available [here](https://ibm.github.io/DPM360/). License, Contribution, Publications are also available there.
+
+<figure><center><img src=./docs/resources/png/dpm360v2.png "DPM360" width="300"/></center><figcaption>DPM360 Component View</figcaption></figure>
+
+# Installation Guides
+
+DPM360 components are interoperable but also work as independent tools. They are working on a cloud cluster (DPM360 server side) and on a client server (DPM360 client side), having separate installation procedures. Please see the guides below to install each component.
+
+## DPM360 server side
+
+For the server side, the <b>installer</b> component sets up an OHDSI stack (Atlas, WebAPI, a Postgres Database, and Achilles) into a cloud cluster such as Kubernetes or OpenShift. See [installation guide](https://github.com/IBM/DPM360/blob/main/installer/docs/installer.md) for details. Its [<b>Express Installation Script</b> section](https://github.com/IBM/DPM360/blob/main/installer/docs/installer.md#express-installation-script) provides minimum setup operations. Using this component:
```

Comment:
Suggested change:

change server side to `One of the key micro-service utilties is <b>installer</b> that does..`